util/envar.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	WatchLogsIntervalSeconds       = "WATCH_LOGS_INTERVAL_SECONDS"
 	DefaultWatchLogsIntervalSecond = 5
+
+	PodIdentifer         = "POD_IDENTIFIER"
```

Comment:
Can you include this in the homepage [README](https://github.com/box/kube-iptables-tailer#environment-variables)?